content/en/users/cloud-compute/auth/_index.md

```diff
@@ -132,29 +135,15 @@ Once you know which project to use, you can use your regular openstack cli
 commands for performing actual operations in the provider:
 
 ```shell
-# Export OIDC env
-export CHECKIN_CLIENT_ID=<CLIENT_ID>
-export CHECKIN_CLIENT_SECRET=<CLIENT_SECRET>
-export CHECKIN_REFRESH_TOKEN=<REFRESH_TOKEN>
-# EGI site
-export EGI_SITE=<NAME_OF_THE_SITE>
-export OS_PROJECT_ID=<PROJECT_ID>
-# get environment variables for openstack
-eval "$(egicli endpoint env)"
-openstack image list
+fedcloud openstack image list --site <NAME_OF_SITE> --vo <NAME_OF_VO>
 ```
 
 For 3rd party tools that can use token based authentication in OpenStack, use
-the following command (after setting the environment as shown above):
+the following command:
 
 ```shell
-# Export OIDC env
-export CHECKIN_CLIENT_ID=<CLIENT_ID>
-export CHECKIN_CLIENT_SECRET=<CLIENT_SECRET>
-export CHECKIN_REFRESH_TOKEN=<REFRESH_TOKEN>
-# EGI site
-export EGI_SITE=<NAME_OF_THE_SITE>
-eval "$(egicli endpoint token)"
+export OS_TOKEN=$(fedcloud openstack --site <NAME_OF_SITE> --vo <NAME_OF_VO> \
+                  token issue -c id -f value)
```

Comment:
Wait for fixing issue 33 https://github.com/tdviet/fedcloudclient/issues/33 to make this command work, a new version should be available on PyPi after weekend

Reply:
Fixed.

Reply:
nice, is this available in PyPi already?

Reply:
Yes, fedcloudclient 1.2.4
https://pypi.org/project/fedcloudclient/

Reply:
In addition, for documentation:
- info about ec3 commands added: https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-ec3-commands
- new guide for using fedcloudclient in scripts added: https://fedcloudclient.fedcloud.eu/scripts.html

Reply:
Thanks! @andrea-manzi, please check if PR looks ok now so we can merge